index.html

```diff
@@ -618,6 +618,93 @@ <h4>
         </section>
       </section>
     </section>
+    <section id="automation" class="informative">
+      <h2>
+        Automation
+      </h2>
+      <p>
+        The Device Posture API pose a challenge to test authors, as fully
+        exercising interface requires physical hardware devices. To address this
+        challenge this document define [[WEBDRIVER2]]
```

Comment:
```suggestion
        challenge this document defines a [[WEBDRIVER2]]
```

Reply:
Fixed in https://github.com/JuhaVainio/W3C-device-posture/pull/1/commits/e8b4c90bfb60199245dae6f40807bf2e6fabd249